apps/video/test/unit/video_test.js

```diff
@@ -13,7 +13,8 @@
   updateSelection,setButtonPaused,handleSliderTouchStart,MockMediaDB,
   handleSliderTouchEnd,loadingChecker,MockL10n,VideoUtils,
   updateDialog,LAYOUT_MODE,showPlayer,hidePlayer,MockThumbnailItem,
-  MockThumbnailGroup,MockForwardRewindController,MockIntlHelper,captions */
+  MockThumbnailGroup,MockForwardRewindController,MockIntlHelper,captions,
+  MockManifestHelper */
```

Comment:
It would also be okay to just remove the manifest related code from the app, since it is not supported anyway. Then you wouldn't have to add this test.